packs/fists-of-the-ruby-phoenix-bestiary/sixth-pillar-student.json

```diff
@@ -2776,7 +2769,38 @@
                     "remaster": false,
                     "title": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "key": "DamageAlteration",
+                        "mode": "override",
+                        "predicate": [
+                            "item:damage:category:energy",
+                            "item:type:spell"
+                        ],
+                        "property": "damage-type",
+                        "selectors": [
+                            "spell-damage",
+                            "inline-damage"
+                        ],
+                        "value": "bludgeoning"
+                    },
+                    {
+                        "itemType": "spell",
+                        "key": "ItemAlteration",
```

Comment:
Wasn't sure whether to create a new passive to hold the air trait ItemAlteration, but decided to put it in here.